src/plugins/crypto/README.md

```diff
@@ -0,0 +1,57 @@
+- infos = Information about crypto plugin is in keys below
+- infos/author = Peter Nirschl <peter.nirschl@gmail.com>
+- infos/licence = BSD
+- infos/needs = 
+- infos/provides = storage
```

Comment:
Sorry, I was thinking about a plugin that encrypts a whole file. In this case filter is good ;)